packages/analytics-browser-test/test/index.test.ts

```diff
@@ -6,7 +6,7 @@ import { default as nock } from 'nock';
 import { success } from './responses';
 import 'isomorphic-fetch';
 import { path, SUCCESS_MESSAGE, url, uuidPattern } from './constants';
-import { PluginType, LogLevel, BaseEvent, Status } from '@amplitude/analytics-types';
+import { LogLevel, BaseEvent } from '@amplitude/analytics-types';
```

Comment:
```suggestion
import type { LogLevel, BaseEvent } from '@amplitude/analytics-types';
```

Reply:
Thanks @kelsonpw. Is it ok if I address this in a later PR?

Reply:
Yes, total nit and can be ignored, I just personally find `type` imports to help readability